assets/src/components/d3/createProgressBar.js

```diff
@@ -56,8 +56,8 @@ function createProgressBar ({ data, width, height, domElement, tip }) {
     .style('outline-color', 'rgb(255, 255, 255)')
```

Comment:
Old assignment planning change. I just did what worked.

Reply:
I'm seeing a console error sometimes that originates on line 50:
```
 Error: <rect> attribute width: Expected length, "NaN".
```

If you change the `attr('width...` part to this, you can see sometimes `NaN` is returned:
```
    .attr('width', d => {
      const result = x(d.percent_gotten) - margin.left
      console.log(result)
      return result
    })
```

Reply:
I got a fix for this and I discussed this with @ssciolla

Reply:
No longer seeing the error 👍